<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.rMk4582' shared service.

return $this->privates['.service_locator.rMk4582'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('categoriesRepository' => function (): \App\Repository\CategoriesRepository {
    return ($this->privates['App\Repository\CategoriesRepository'] ?? $this->load('getCategoriesRepositoryService.php'));
}, 'detailRepository' => function (): \App\Repository\OrderDetailRepository {
    return ($this->privates['App\Repository\OrderDetailRepository'] ?? $this->load('getOrderDetailRepositoryService.php'));
}, 'order' => function (): \App\Entity\Orders {
    return ($this->privates['.errored..service_locator.rMk4582.App\Entity\Orders'] ?? $this->load('getOrders3Service.php'));
}, 'shopcartRepository' => function (): \App\Repository\ShopcartRepository {
    return ($this->privates['App\Repository\ShopcartRepository'] ?? $this->load('getShopcartRepositoryService.php'));
}));
